### ISSUE TITLE
startLines in tag-lines breaks when there is only a description

### ISSUE THREAD

**Issue by stof** [gajus/eslint-plugin-jsdoc#1024]
## Expected behavior
A JSDoc with only a description should not be reported as an error.

## Actual behavior
An error is reported asking for a newline after the block description, while this is the end of the jsdoc

## ESLint Config

```json
{
    "plugins": ["jsdoc"],
    "rules": {
        "jsdoc/tag-lines": ["error", "never", {
            "startLines": 1
        }],
    }
}
```

## ESLint sample

```js

    /**
     * Toggles the deselect all icon button action
     */
    function updateIconButton () {
        // <redacted>
    }
```

## Environment

- Node version: 18.16.0
- ESLint version: 8.38.0
- `eslint-plugin-jsdoc` version: 43.0.1
